apps/next/app/dashboard/page.tsx

```diff
@@ -45,6 +46,53 @@ async function findCreateUserWithClerkId({
   return newUser;
 }
 
+const ensureTeamForClerkOrg = async ({
+  clerkOrgId,
+  teamName,
+  userId,
+}: {
+  clerkOrgId: string;
+  teamName: string;
+  userId: string;
+}): Promise<void> => {
+  /* Fixes an issue where if a user already has a Metal account and then uses
+  Clerk's <OrganizationSwitcher /> to create a new org, the new Clerk org will
+  exist but won't have an associated Metal team. This function will create a
+  team for the  and then add the user to it.
+  */
+  let usersPersonalTeam = await db.query.teams
+    .findMany({
+      where: (team, { eq, and }) =>
+        and(eq(team.clerkId, clerkOrgId), eq(team.creatorId, userId)),
```

Comment:
i don't think the userId condition is necessary--we're just trying to create a team that corresponds with a clerk org

Reply:
Removed.